Modules/Libs/QuestieLib.lua

```diff
@@ -535,3 +535,114 @@ function QuestieLib.tunpack(tbl)
+        local lines = {}
+        local startIndex = 1
+        local endIndex = 2 --We should be able to start at a later index...
+        --This function returns a list of size information per row, so we use this to calculate number of rows
+        local numberOfRows = #objectiveFontString:CalculateScreenAreaFromCharacterSpan(startIndex, strlen(useLine))
+        --print("lineLen", strlen(line))
+        for row = 1, numberOfRows do
+            local lastSpaceIndex
+            local dotIndex
+            local indexes
+            --We use the previous way to get number of rows to loop through characterindex until we get 2 rows
+            repeat
+                indexes = objectiveFontString:CalculateScreenAreaFromCharacterSpan(startIndex, endIndex)
+                --Last space of the line to be used to break a new row
+                if(string.sub(useLine, endIndex, endIndex) == " ") then
+                    lastSpaceIndex = endIndex
+                --Track the dot at the end of a line
+                elseif(string.sub(useLine, endIndex, endIndex) == "." and endIndex ~= strlen(useLine) and splitOnDot) then
+                    dotIndex = endIndex
+                end
+                endIndex = endIndex + 1
+                --If we are at the end of characters break and set endIndex to strlen
+                if(endIndex > strlen(useLine)) then
+                    endIndex = strlen(useLine)
+                    lastSpaceIndex = endIndex
+                    break
+                end
+            until (#indexes > 1) --Until more than one row
+
+            --Get the line we calculated
+            --First to Dot, then space and lastly endIndex(chinese)
+            local newLine = string.sub(useLine, startIndex, dotIndex or lastSpaceIndex or endIndex)
+
+            --This combines a trailing word to the previous line if it is the only word of the line
+            --We check lastSpaceIndex here because the logic will be faulty (chinese client)
+            if(row == numberOfRows-1 and combineTrailing and lastSpaceIndex) then
+                --Get the last line, in it's full
+                local lastLine = string.sub(useLine, endIndex - 2, strlen(useLine))
+
+                --Does the line not contain any space we combine it into the previous line
+                if(not string.find(lastLine, " ")) then
+                    newLine = string.sub(useLine, startIndex, strlen(useLine))
+                    --print("NL1", newLine)
+                    table.insert(lines, prefix..newLine)
+                    --Break the for loop on last line, no more running required
+                    break
+                end
+            end
+            --Change the startIndex to be the new line, and add the line to the lines list
+            startIndex = endIndex - 2
+            endIndex = endIndex
+
+            table.insert(lines, prefix..newLine)
+        end
+        --Hide the fontstring
+        objectiveFontString:Hide()
```

Comment:
```suggestion
        objectiveFontString:Hide()
```

This is quite good to read, no need for a comment